lib/commands/base.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Commands
+  class Base
+    attr_reader :bot, :chat_id
+
+    def initialize(bot:, chat_id:)
+      @bot = bot
```

Comment:
In all classes you use `bot.api`, you don't use any other methods, that's why will be better if you send in classes only `api` object